lstchain/scripts/lstchain_create_run_summary.py

```diff
@@ -0,0 +1,216 @@
+    """Get the list of R0 files from a given date."""
+    path_r0 = args.R0_PATH / date
+    # FIXME: use regular expressions from lstchain.paths.R0_RE
+    list_of_files = path_r0.glob("LST*.fits.fz")
+    return list_of_files
+
+
+def get_list_of_runs(list_of_files):
+    """Get the sorted list of run objects from R0 filenames."""
+    return sorted(parse_r0_filename(file) for file in list_of_files)
+
+
+def get_run_numbers(list_of_runs):
+    """Get the list of run numbers of a given date."""
+    return np.unique([run_info.run for run_info in list_of_runs])
+
+
+def get_number_of_subruns(run_number, list_of_run_objects):
+    """Obtain the number of sequential files (subruns) of a given run."""
+    filtered_run = filter(lambda x: x.run == run_number, list_of_run_objects)
+    last_subrun = max(list(filtered_run), key=lambda x: x.subrun)
+    return last_subrun.subrun
+
+
+def start_of_run_datetime(run_number):
+    """
+    Get datetime of the start of the run in ISOT format and UTC scale.
+    """
+    # TODO: Use pathlib.stat_results instead to get creation file timestamp instead
+    filename = args.R0_PATH / args.date / f"LST-1.1.Run{run_number:05d}.0000.fits.fz"
+    file = protozfits.File(str(filename))
+    return file.CameraConfig.header["DATE"]
+
+
+def type_of_run(run_number, n_events=500):
+    """
+    Get empirically the type of run based on the percentage of
+    pedestals/mono trigger types from the first 10000 events (?):
+    100% mono events (trigger 1): DRS4 pedestal run
+    <10% pedestal events (trigger 32): cosmic DATA run
+    Otherwise (~50% mono, ~50% pedestal events): PEDESTAL-CALIBRATION run
+    First subrun needs to be open.
+    """
+    filename = args.R0_PATH / args.date / f"LST-1.1.Run{run_number:05d}.0000.fits.fz"
+
+    with EventSource(input_url=filename, max_events=n_events) as source:
+        n_pedestal = sum(
+            1 for event in source if event.trigger.event_type == EventType.SKY_PEDESTAL
+        )
+        n_sky = sum(
+            1 for event in source if event.trigger.event_type == EventType.SUBARRAY
+        )
+
+    # FIXME: Do this classification in some other way?
+    if n_sky / n_events > 0.999:
+        run_type = "DRS4"
+    elif n_pedestal / n_events > 0.1:
+        run_type = "CALI"
+    elif n_pedestal / n_events < 0.1:
+        run_type = "DATA"
+    else:
+        run_type = "UNKW"
+
+    return run_type
+
+
+def get_initial_timestamps(run_number):
+    """
+    Get initial valid timestamps. Write down the Dragon module used.
+    First subrun needs to be open. Taken from ctapipe_io_lst.event_time
+    """
+    filename = args.R0_PATH / args.date / f"LST-1.1.Run{run_number:05d}.0000.fits.fz"
+
+    with EventSource(input_url=filename) as source:
+
+        for event in source:
+            lst = event.lst.tel[tel_id]
+            central_module_index = np.where(lst.svc.module_ids == CENTRAL_MODULE)[0][0]
+            module_index = central_module_index
```

Comment:
@FrancaCassol made the proposal to use the first available module that has `module_status != 0`

Reply:
I will implement that

Reply:
Done. I'm now getting: `module_index = np.where(lst.evt.module_status != 0)[0][0]`